seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/source/reader/ParquetReadStrategy.java

```diff
@@ -100,9 +98,6 @@ public void read(String path, String tableId, Collector<SeaTunnelRow> output)
                                 HadoopInputFile.fromPath(filePath, configuration));
         int fieldsCount = seaTunnelRowType.getTotalFields();
         GenericData dataModel = new GenericData();
-        dataModel.addLogicalTypeConversion(new Conversions.DecimalConversion());
```

Comment:
Why delete this? This will cause abnormal data to be read out.

Reply:
Thanks, this was a mistake.

Reply:
reverted